controllers/feeds/getFeedChat.js

```diff
@@ -3,6 +3,7 @@ const {Sequelize} = require('sequelize');
 const UsersFunction = require('../../databases/functions/users');
 const {User, sequelize} = require('../../databases/models');
 const getstreamService = require('../../services/getstream');
+const roundingKarmaScore = require('../../helpers/roundingKarmaScore');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of the `roundingKarmaScore` helper function is a good addition. It will help in rounding off the karma scores before they are used or displayed, ensuring consistency across the application.
```diff
+ const roundingKarmaScore = require('../../helpers/roundingKarmaScore');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->